source/common/http/conn_manager_impl.cc

```diff
@@ -439,6 +447,17 @@ void ConnectionManagerImpl::ActiveStream::onIdleTimeout() {
   }
 }
 
+void ConnectionManagerImpl::ActiveStream::onRequestTimeout() {
+  connection_manager_.stats_.named_.downstream_rq_path_timeout_.inc();
+  if (response_headers_ != nullptr) {
```

Comment:
Comment describing the semantic meaning of these tests?  I.e. something like "If the upstream has started to respond, just close the connection.  Otherwise, send a response indicating request timeout."  (I don't guarantee I have that right, just giving an example of the kind of comment I'd like to see.)

Reply:
Actually, followup on this: Do you have any tests that test these conditionals?

Reply:
Looked at this for a bit and realized the codepath doesn't need get hit in any condition. Removed.